packages/mobile/src/app/WebViewScreen.tsx

```diff
@@ -22,13 +20,7 @@ export const webViewScreenNavOptions = ({ route }: RouteProps) => {
   return {
     ...emptyHeader,
     headerTitle: hostname || ' ',
-    headerLeft: () => (
-      <TopBarTextButton
-        title={i18n.t('global:close')}
-        onPress={navigateBack}
-        titleStyle={styles.close}
-      />
-    ),
+    headerLeft: () => <BackButton onPress={navigateBack} />,
```

Comment:
I'll upload images of the default screens similar to how I had in #500.

Having a style a guide is a great idea. Could that live in this repo?

Reply:
@coreycelo can you provide feedback on this?

Reply:
After a design review with @coreycelo. The back button remains an icon and the close is text for default font sizes. If larger text is enabled it will be an 'x' icon.